README.md

```diff
@@ -1,67 +1,83 @@
-# Overview
-
-A simple and efficient blockchain ledger.
-
-## About
+# Hyperledger Iroha
 
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 ![Rust](https://github.com/hyperledger/iroha/workflows/Rust/badge.svg?branch=iroha2-dev)
 [![codecov](https://codecov.io/gh/hyperledger/iroha/branch/iroha2-dev/graph/badge.svg)](https://codecov.io/gh/hyperledger/iroha)
 
+Iroha is a simple and efficient blockchain ledger based on the **distributed ledger technology (DLT)**. Its design principles are inspired by the Japanese Kaizen principle of eliminating excesses (*muri*).
 
-Iroha is a distributed ledger technology (DLT). Its design principles are inspired by the Japanese Kaizen principle — eliminate excesses (muri). Iroha can help you manage your accounts, assets, on-chain data storage with efficient smartcontracts, while being Byzantine- and crash-fault tolerant, being able to withstand a fault rate of 33%.
+Iroha can help you manage your accounts, assets, on-chain data storage with efficient smart contracts, while being Byzantine- and crash-fault tolerant with a fault rate of **33%**.
 
 ## Features
 
-Iroha is a fully-featured blockchain ledger
+Iroha is a fully-featured blockchain ledger. With Iroha you can:
+
+* Create and manage custom fungible assets, such as currencies, gold, and others
+* Create and manage non-fungible assets
+* Manage user accounts with a domain hierarchy and multi-signature transactions
+* Use efficient portable smart contracts implemented either via WebAssembly or Iroha Special Instructions
+* Use both permissioned and permission-less blockchain deployments
+
+Iroha offers:
 
-* Creation and management of custom fungible assets, such as currencies, gold, etc.
-* Non-fungible asset support.
-* User account management, with a domain hierarchy, and multi-signature transactions.
-* Efficient portable smartcontracts implemented either via WebAssembly, or Iroha Special Instructions.
-* Support for both permissioned, and permission-less blockchain deployments.
-* Byzantine fault-tolerance with up to 34% fault rate.
-* Efficient in-memory operation.
-* Extensive telemetry support out of the box.
-* Modular structure. Don't need it? Compile with `--no-default-features` and add the features you need.
-* Event-driven architecture with strongly-typed events.
+* Byzantine fault-tolerance with up to 34% fault rate  
```

Comment:
@appetrosyan not sure why the numbers here and on line 9 are different

Reply:
They're not the actual number is `1/3` which some people rounded up, and some-down. I think we should omit the figure on line 9 and replace this with `33%`, because `1/3` is only accurate for large numbers of peers.